umbraco/Umbraco-CMS
# V9: Display configured log levels

## Details
Fixes: #11626
- If UmbracoFile sink is not configured - the log level box should only display the global minimum (it looks very much like what we have currently):
![image](https://user-images.githubusercontent.com/21998037/144294020-075ef76c-8214-4e87-b2ca-622348edf6ca.png)

- If **it is**, we will get both levels:
![image](https://user-images.githubusercontent.com/21998037/144293774-17d42a3a-0541-42b0-87c7-136c6aa0a2e8.png)

- The old `GetLogLevel()` is obsoleted.

## Test:
- Run client project;
- Modify the `Serilog:MinimumLevel:Default` level in the _appsettings.Development.json_ configuration file, for example;
- Make sure the changes are reflected;
- Add [Umbraco sink](https://our.umbraco.com/documentation/reference/V9-Config/Serilog/#changing-the-umbraco-sink);
- Verify the level is displayed;
